backend/StreamServerApp/subtitles.py

```diff
@@ -16,21 +16,49 @@ def init_cache():
             'filename': 'cachefile.dbm'}, replace_existing_backend=True)
 
 
-def handle_subliminal_download(video, video_path, langage):
-    best_subtitles = download_best_subtitles([video], {Language(langage)})
+def handle_subliminal_download(video, video_path):
+    """ # Download the best subtitles in french and english
+        Args:
+        video : Name of video
+        video_path: absolute path to videos
+    """
+    best_subtitles = download_best_subtitles([video], {Language('fra'),Language('eng')})
```

Comment:
Probably we could get better code readability and modularity by defining languages beforehand and avoiding repeating the definition of them.

We could start by:
```python
languages_to_retrieve = {   # this kind of things should ideally be on a config file like the 
    Language('eng'),        # django settings file, but it's ok to keep it here for now
    Language('fra'),
}

best_subtitles = download_best_subtitles([video], languages_to_retrieve)
```